ui/src/data_explorer/components/QueryMaker.js

```diff
@@ -3,6 +3,7 @@ import React, {PropTypes} from 'react'
 import QueryBuilder from './QueryBuilder'
 import QueryMakerTab from './QueryMakerTab'
 import buildInfluxQLQuery from 'utils/influxql'
+import classNames from 'classnames'
```

Comment:
preferred convention is to use lowercase `classnames` -- to me this is preferable also, as it helps differentiate clearly between React's `className` special prop and the library function `classnames` we're using